package.json

```diff
@@ -25,6 +25,9 @@
     "ng": "nx",
     "nx": "nx",
     "start": "npm-run-all --parallel \"ng serve\" \"build:core:watch\"",
+    "start:storybook": "npm run compodoc:designsystem && npm run bundle-storybook-assets && ng run designsystem:storybook",
+    "bundle-storybook-assets": "mkdir -p libs/designsystem/.storybook/static/assets/kirby/icons/svg && cp libs/designsystem/src/lib/icons/svg/** libs/designsystem/.storybook/static/assets/kirby/icons/svg ",
```

Comment:
We should find a better way of doing this https://github.com/kirbydesign/designsystem/issues/1879. But this can act as a temporary solution for now.